app/controllers/ops_controller/ops_rbac.rb

```diff
@@ -845,8 +859,9 @@ def process_roles(roles, task)
     process_elements(roles, MiqUserRole, task)
   end
 
-  def process_tenants(tenants, task)
-    process_elements(tenants, Tenant, task, "Tenant", "name")
+  def process_tenants(tenants, task, display_model = 'Tenant')
```

Comment:
ok. Still think keeping the whole list of tenants together is preferred.
Then `partition` it in `process_tenants` -- see the code [here](https://github.com/ManageIQ/manageiq/pull/6264/files#r50338849)